src/utils/DynamicContent/DynamicContent.ts

```diff
@@ -3,6 +3,7 @@ import { DynamicProgressBarContent } from '@src/utils/DynamicContent/DynamicProg
 export interface DynamicContent {
 	currentDayName: string;
 	currentDate: string;
+	currentDateAndTime: () => string;
```

Comment:
It would be cool if you could document the difference between properties and getter methods in the interface docbloc. Also maybe call this `getCurrentDateAndTime()`